.github/workflows/ci.yaml

```diff
@@ -0,0 +1,126 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+
+    - uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key:
+          ${{ runner.os }}-pip-lint-${{ hashFiles('pyproject.toml', 'setup.py', 'tox.ini') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-lint-
+          ${{ runner.os }}-pip-
+
+    - run: python -m pip install tox
+
+    - run: tox -q -e flake8
```

Comment:
maybe run all 4 in parallel ? maybe it's faster.

```suggestion
    - run: tox -q -p all -e flake8, towncrier, twine, check-manifest
```

I think that in the future, this can be extended into an automated release to pypi job.